entities/markets/market.ts

```diff
@@ -2,20 +2,31 @@ import { FixedProductMarketMaker } from '@/queries/omen';
 import { fromHex } from 'viem';
 import { Outcome } from '@/entities';
 import { isPast } from 'date-fns';
+import { _24HoursInSeconds, nowTimestamp } from '@/utils/time';
```

Comment:
```suggestion
import { _24HoursInSeconds, nowTimestamp } from '@/utils';
```

Reply:
I had what you suggest, but had to change since it was giving me errors. I suggest you to try to actually see what's going on. This is what I don't like about NextJS :)

Reply:
what kind of errors? 

usually it works like this.

Reply:
Strange ones, like "ReferenceError: Cannot access 'c' before initialization". I suggest you to try it.